stack/indexer/base/builder.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+set -x
```

Comment:
This should be removed as a debug element

Reply:
It's good to include build information, in builders doesn't mind the additional amount of information,